docs/source/deployguide/cadeploy.md

```diff
@@ -0,0 +1,597 @@
+# CA Deployment steps
+
+## Download the binaries
+
+The Fabric CA server and CA client binaries are available for download from [github](https://github.com/hyperledger/fabric-ca/releases). Scroll down to **Assets** and select the latest binary for your machine. The .zip file contains both the CA server and the CA client binaries. Copy the CA server file to a directory where you will run the server. Likewise, copy the CA client binary to its own directory. Having the CA client in its own folder facilitates certificate management especially when you need to interact with multiple CAs.  When you issue a command from the CA client against a CA server, you can target a specific CA by modifying the CA server URL on the request. Therefore, only a single Fabric CA client binary is required and can be used to transact with multiple CAs. More on using the Fabric CA client below.
+
+## Fabric CA client
+
+While you can use the Fabric SDKs to interact with your CA, it is recommended that you **use the Fabric CA client to register and enroll node admin identities**. The instructions provided in this topic assume a single Fabric CA client is being used.
+
+**Tip:** While you can run the Fabric CA client binary from anywhere you want, for ease of following these instructions we will refer to it in its own directory named `fabric-ca-client`.
+
+Because you will use the CA client to submit transactions to the CA for enrolling identities, certificate management is critically important when using the CA client. The private and public keys are first generated locally by the Fabric CA client, and then the public key is sent to the CA which returns an encoded "signed certificate". A best practice therefore is for each TLS CA, organization CA, or intermediate CA that you create, to create the following folders:
+- You need a sub-folder in order to connect to each CA, such as `/tls-ca` or `/org1-ca` or `/int-ca`. This folder can be under the Fabric CA client or anywhere that the CA client can access the path. For purposes of these instructions, these folders reside inside the `fabric-ca-client` directory. For example:
+
+  ```
+  mkdir fabric-ca-client
+  cd fabric-ca-client
+  mkdir tls-ca org1-ca int-ca
+  ```
+- Under each CA folder, you need a folder to store the TLS certificate, such as `org1-ca/tls`. The TLS certificate secures the communications between the CA client and the CA server. Secondly, you need to create a folder to place the generated certificates on an enroll command. If this is an admin identity, it would be  `org1-ca/admin`.
```

Comment:
Where to get the TLS cert from?
I was expecting this doc would first talk about CA server setup, and THEN talk about how to connect the CA client to the servers for registrations/enrollments.

Actually... I see the answer on line 183 when CA client is setup.  Not sure why we talk about this setup here and then again on line 183. Makes more sense down at line 183.

Reply:
The progressive disclosure is along these lines:
- You download the binaries.
- There are 2 of them:  server and client
- I talk about the server binaries
- Then I talk about the client binaries. Since I introduce the client binaries, I go ahead and have them set up the folder structure for what they need later. It felt more straightforward to set up all these folders here and now, so that the during the rest of the tutorial, everything is in place and ready and they are not jumping around.

I looked at moving this entire discussion to later in the topic - after you create the first CA for example, but it's quite distracting and ruins the  init CA->modify cfg->start CA->enroll bootstrap identity pattern that we promote. I'll continue to think about this organization though to see if there is better way.

In the meantime, I changed it to 
```Under each CA folder, you need a folder to store the root TLS certificate, such as `org1-ca/tls`. The TLS certificate secures the communications between the CA client and the CA server. This certificate, `ca-cert.pem` is generated after TLS is enabled on the TLS CA and it is subsequently started.```